depthai_helpers/arg_manager.py

```diff
@@ -50,14 +54,18 @@ def parse_args():
                         help="Disparity confidence threshold, used for depth measurement. Default: %(default)s")
     parser.add_argument("-med", "--stereo_median_size", default=7, type=int, choices=[0, 3, 5, 7],
                         help="Disparity / depth median filter kernel size (N x N) . 0 = filtering disabled. Default: %(default)s")
+    parser.add_argument("-maxd", "--max-disparity", default=95, type=int,
```

Comment:
Shouldn't be parameter since it's not configurable by the user.
Should be decided based on the stereo mode enabled, since we don't have API to retrieve it, but @alex-luxonis proposed it.
See here the max disparity range: https://github.com/luxonis/depthai-experiments/blob/master/gen2-camera-demo/main.py#L189